### DISCUSSION TITLE
Errors in debug logs from configuration of the DNA plugin

### DISCUSSION THREAD

**Discussion by fostermi** [389ds/389-ds-base#6135] [Q&A] [answered]
I've enabled the Distributed Numeric Plugin and configured it according to this document: https://www.port389.org/docs/389ds/design/dna-plugin.html, but I see two ERR messages in the debug as soon as I apply the config for unique numbers.

```
[25/Mar/2024:21:08:53.862388352 +0000] - ERR - slapi_search_internal_set_pb - NULL parameter
[25/Mar/2024:21:08:53.863642252 +0000] - ERR - allow_operation - Component identity is NULL
```
This happens on a fresh install of 389ds in Docker or on a legacy system. From what I can tell, it doesn't affect the server at all and everything is working. 

The ldif for enable the configuration looks like this:
```
dn: cn=UID and GID numbers,cn=Distributed Numeric Assignment Plugin,cn=plugins,cn=config
changetype: add
objectClass: extensibleObject
objectClass: top
cn: UID and GID numbers 
dnaFilter: (|(objectclass=posixAccount)(objectclass=posixGroup))
dnaMagicRegen: -1
dnaMaxValue: 200000
dnaNextValue: 100002
dnaRangeRequestTimeout: 60
dnaScope: dc=example,dc=com
dnaThreshold: 100
dnaType: uidNumber
dnaType: gidNumber
```

Any ideas on how to further troubleshoot which part of the configuration 389ds is complaining about?

**Comment by mreynolds389** [comment, marked as answer]
What version of 389-ds-base are you using?  

I'm pretty sure we cleaned these error messages up in newer versions (because they are harmless).  Then again there could be a couple of places where these messages are still logged that we didn't clean it up.

Either way, it's nothing to be alarmed about and you can ignore them for now.

**Reply by fostermi** [reply]
Ah, thanks for the info. The version I'm using is 3.0.

**Reply by mreynolds389** [reply]
Ok that is a recent version, so it looks like we have more logging to cleanup...
